README.md

```diff
@@ -0,0 +1,58 @@
+# flatpak - org.onlyoffice.desktopeditors
+
+## Overview
+
+ONLYOFFICE Desktop Editors is a free office suite that combines text,
+spreadsheet and presentation editors allowing to create, view and edit
+documents stored on your Windows/Linux PC or Mac without an Internet connection.
+It is fully compatible with Office Open XML formats: .docx, .xlsx, .pptx.
+
+## Quick Setup
+
+### Install via flathub.org
+
+1. Install flatpak runtime on your system according to [instruction](https://flatpak.org/setup/)
+2. Install application via
+
+   ```shell script
+   flatpak install flathub org.onlyoffice.desktopeditors
+   ```
+
+3. Rum the application
+
+    ```shell script
+   flatpak run org.onlyoffice.desktopeditors
+   ```
+
+### Build application manually
+
+All steps tested on Ubuntu 18.04
+
+1. Clone this repo:
+
+   ```shell script
+   git clone https://github.com/flathub/org.onlyoffice.desktopeditors
+   cd org.onlyoffice.desktopeditors
+   ```
+
+2. Install `flatpak` and `flatpak-builder` according to [instruction](https://flatpak.org/setup/)
+
+3. Install a runtime and the matching SDK
+
+   ```shell script
+   flatpak install flathub org.freedesktop.Platform//19.08 \
+                   org.freedesktop.Sdk//19.08 \
+                   io.atom.electron.BaseApp//19.08
+   ```
+
+4. Build and install the application
+
+   ```shell script
+   sudo flatpak-builder build org.onlyoffice.desktopeditors.json --force-clean --install
```

Comment:
Never run flatpak with sudo. It uses polkit to ask for privileges when they are needed

Reply:
Thanks, fixed it